blaze/expr/arithmetic.py

```diff
@@ -220,7 +221,7 @@ def scalar_coerce(_, val):
 
 @dispatch(ct.DateTime, _strtypes)
 def scalar_coerce(_, val):
-    return dt_parse(val)
+    return pd.Timestamp(val)
```

Comment:
In what sense?

```python
In [26]: import sqlalchemy as sa

In [27]: c = sa.Column('col', sa.types.TIMESTAMP)

In [28]: expr = c >= pd.Timestamp('now').to_pydatetime()

In [29]: print(expr.compile().render_literal_bindparam(expr.right))
```
raises `NotImplementedError`, and so does the version with the `Timestamp` instance.